proto/mixer.proto

```diff
@@ -422,8 +422,8 @@ service Mixer {
   rpc BulkObservationsPoint(datacommons.v1.BulkObservationsPointRequest)
       returns (datacommons.v1.BulkObservationsPointResponse) {
     option (google.api.http) = {
-      get : "/v1/bulk/observations"
-      additional_bindings : {post : "/v1/bulk/observations" body : "*"}
+      get : "/v1/bulk/observations/point"
+      additional_bindings : {post : "/v1/bulk/observations/points" body : "*"}
```

Comment:
"get" uses "point", but post uses "points" -- shall we be consistent between "get" and "post"?

Reply:
Good catch! Changed